README.md

```diff
@@ -82,16 +81,16 @@ For more details, see [pyproject.toml](python/rmm/pyproject.toml)
 
 To install RMM from source, ensure the dependencies are met and follow the steps below:
 
-- Clone the repository and submodules
+- Clone the repository
 ```bash
-$ git clone --recurse-submodules https://github.com/rapidsai/rmm.git
+$ git clone https://github.com/rapidsai/rmm.git
 $ cd rmm
 ```
 
 - Create the conda development environment `rmm_dev`
 ```bash
 # create the conda environment (assuming in base `rmm` directory)
-$ conda env create --name rmm_dev --file conda/environments/all_cuda-118_arch-x86_64.yaml
+$ conda env create --name rmm_dev --file conda/environments/all_cuda-125_arch-x86_64.yaml
```

Comment:
Is `--name` even needed? It should be in the .yaml, right?

Reply:
The name in the environment is not as user-friendly. Overriding it is a better UX.